### DISCUSSION TITLE
What dose 'OR Crystalloid Intake' mean ?

### DISCUSSION THREAD

**Discussion by pike8801** [MIT-LCP/mimic-code#1193] [MIMIC-IV] [answered]
I would like to select the fluidcategory for fluid resuscitation， in the ‘mimic_icu.d_items’ , some labels confuse me, such as 'or crystalloid intake',' PACU crystalloid intake', 'Gu irigant - normal saline',' Po intake', 'GT flush'. Can anyone explain to me,Thank you!

**Comment by alistairewj** [comment, marked as answer]
Operating Room crystalloid intake. It's documentation of pre-admission fluid intake. PACU (post-anesthesia care unit) is similar. The other items are different - GU irrigant is the use of irrigation, PO intake is per oral intake (i.e. they drank something), GT flush is a tube flush. These are not used for resuscitation.
